.issues/20260904_버그_같은_모집글_링크_재탭_시_콜드_스타트_홈_이동.md

```
@@ -0,0 +1,66 @@
+# ❗ [버그][Flutter][딥링크] 같은 모집글 링크를 다시 누르면 콜드 스타트에서 홈으로 이동함
```

markdownlint-cli2 (0.23.2)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "딥링크"

(MD052, reference-links-images)